09_bg_change/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
I think you created this branch from 09. That's why these files are here too.

For now, keep them here, but next time don't forget to create the branch from master!

Reply:
Hm it happened again with my 11_ folder. I did create from master... not sure why it happens...